lib/flutter/flutter_sdk.dart

```diff
@@ -44,15 +46,17 @@ class FlutterSdkManager implements Disposable {
       .transform(new Debounce(new Duration(seconds: 1)))
       .listen((value) => _setSdkPath(value));
 
-    _disposables.add(atom.commands.add('atom-workspace', 'flutter:auto-locate-flutter-sdk', (_) {
-      new SdkLocationJob(this).schedule();
-    }));
-    _disposables.add(atom.commands.add('atom-workspace', 'flutter:show-flutter-sdk-info', (_) {
-      showInstallationInfo();
-    }));
-    _disposables.add(atom.commands.add('atom-workspace', 'flutter:version', (_) {
-      showVersionInfo();
-    }));
+    if (Flutter.hasFlutterPlugin()) {
```

Comment:
Not really, it's just that they are visible from the command palette, which will be odd, when they don't have the flutter plugin installed.